projects/1.16/assets/mystical-world/mysticalworld/lang/zh_cn.json

```
@@ -1 +1,434 @@
+    "block.mysticalworld.uncanny_mushroom_slab": "魔幻菇台阶",
+    "block.mysticalworld.uncanny_mushroom_small_post": "魔幻菇窄柱",
+    "block.mysticalworld.uncanny_mushroom_stairs": "魔幻菇楼梯",
+    "block.mysticalworld.uncanny_mushroom_wall": "魔幻菇墙",
+    "block.mysticalworld.uncanny_mushroom_wide_post": "魔幻菇宽柱",
+    "block.mysticalworld.uncanny_sand": "魔幻沙",
+    "block.mysticalworld.wet_mud_block": "湿泥块",
+    "block.mysticalworld.wet_mud_brick": "湿泥砖",
+    "block.mysticalworld.wild_aubergine": "野生茄子",
+    "block.mysticalworld.wild_wart": "野生疣",
+    "effect.mysticalworld.serendipity": "意外惊喜",
+    "effect.mysticalworld.slow_regeneration": "减缓再生",
+    "effect.mysticalworld.wakeful": "失眠",
+    "entity.mysticalworld.beetle": "甲虫",
+    "entity.mysticalworld.clam": "蛤蜊",
+    "entity.mysticalworld.deer": "鹿",
+    "entity.mysticalworld.duck": "鸭",
+    "entity.mysticalworld.endermini": "迷你末影人",
+    "entity.mysticalworld.frog": "青蛙",
+    "entity.mysticalworld.hell_sprout": "下界芽精",
+    "entity.mysticalworld.lava_cat": "熔岩猫",
+    "entity.mysticalworld.owl": "猫头鹰",
+    "entity.mysticalworld.silkworm": "蚕",
+    "entity.mysticalworld.silver_fox": "银狐",
+    "entity.mysticalworld.sprout": "芽精",
+    "item.minecraft.lingering_potion.effect.serendipity": "滞留型意外惊喜药水",
+    "item.minecraft.potion.effect.serendipity": "意外惊喜药水",
+    "item.minecraft.splash_potion.effect.serendipity": "喷溅型意外惊喜药水",
+    "item.minecraft.tipped_arrow.effect.serendipity": "意外惊喜之箭",
+    "item.mysticalworld.antler_hat": "鹿角帽",
+    "item.mysticalworld.antlers": "鹿角",
+    "item.mysticalworld.apple_cordial": "苹果果汁",
+    "item.mysticalworld.assorted_seeds": "混杂的种子",
+    "item.mysticalworld.aubergine": "茄子",
+    "item.mysticalworld.aubergine_salad": "茄子沙拉",
+    "item.mysticalworld.aubergine_seeds": "茄子种子",
+    "item.mysticalworld.beetle_boots": "甲虫靴子",
+    "item.mysticalworld.beetle_chestplate": "甲虫胸甲",
+    "item.mysticalworld.beetle_helmet": "甲虫头盔",
+    "item.mysticalworld.beetle_leggings": "甲虫护腿",
+    "item.mysticalworld.beetle_spawn_egg": "甲虫刷怪蛋",
+    "item.mysticalworld.beetroot_salad": "甜菜根沙拉",
+    "item.mysticalworld.cactus_axe": "仙人掌斧",
+    "item.mysticalworld.cactus_dandelion_salad": "仙人掌蒲公英沙拉",
+    "item.mysticalworld.cactus_hoe": "仙人掌锄",
+    "item.mysticalworld.cactus_knife": "仙人掌刀",
+    "item.mysticalworld.cactus_pickaxe": "仙人掌镐",
+    "item.mysticalworld.cactus_shovel": "仙人掌锹",
+    "item.mysticalworld.cactus_spear": "仙人掌矛",
+    "item.mysticalworld.cactus_sword": "仙人掌剑",
+    "item.mysticalworld.cactus_syrup": "仙人掌果汁",
+    "item.mysticalworld.carapace": "甲壳",
+    "item.mysticalworld.clam_bucket": "蛤蜊桶",
+    "item.mysticalworld.clam_spawn_egg": "蛤蜊刷怪蛋",
+    "item.mysticalworld.cooked_aubergine": "烤茄子",
+    "item.mysticalworld.cooked_beetroot": "烤甜菜根",
+    "item.mysticalworld.cooked_carrot": "烤胡萝卜",
+    "item.mysticalworld.cooked_seeds": "烤种子",
+    "item.mysticalworld.cooked_squid": "烤鱿鱼",
+    "item.mysticalworld.cooked_venison": "烤鹿肉",

```

```suggestion
    "item.mysticalworld.cooked_venison": "熟鹿肉",
```
和原版肉类统一
其他几个cooked我也不知道要怎么处理（），不过现在这样就挺好的